apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1977,8 +1977,14 @@ contract_transactions(_Config) ->    % miner has an account
     PoI = aec_trees:deserialize_poi(PoIBin),
     {ok, ContractInPoI} = aec_trees:lookup_poi(contracts, ContractPubKey, PoI),
     {ok, Trees} = rpc(aec_chain, get_top_state, []),
-    ContractInPoI = rpc(aect_state_tree, get_contract, [ContractPubKey,
+    ContractInPoI1 = rpc(aect_state_tree, get_contract, [ContractPubKey,
                                                          aec_trees:contracts(Trees)]),
+    %% Don't require the store mp trees to be identical, but check that the
```

Comment:
MPT is identical if the content is the same. That is the whole point of them. If that test fails, something is wrong.

Reply:
Yes, this one had me a bit worried.

In a lot of places we compare stores with a non-empty write cache, to a store obtained from state trees (with values stored in the MPT). I think this test failed with different MPTs though.

I will do some investigation.